### PR TITLE
Added data/front-matter.md which is added to the generated HTML catalog.  

### PR DESCRIPTION
**Pull request by apannetrat**
The generated HTML catalog did not contain the names of contributors and only limited introductory information for the catalog. 
With this change, we can edit the markdown file data/front-matter.md, and its content will automatically be incorporated into the generated HTML catalog.

### CODE REVIEW

File: tools/metrics_validator.rb [hunk 1]
```diff
@@ -73,12 +73,13 @@ def render(doc, data)
   end
 end
 
-if ARGV.length!=2
-  puts "Usage: #{__FILE__} <source_yaml_file> <html_rending>"
+if ARGV.length!=3
+  puts "Usage: #{__FILE__} <source_yaml_file> <souce_makrdown_front_matter> <html_rending>"
```
Comment (pritikin) [inline tools/metrics_validator.rb:77]: does this typo matter?

**Review by pritikin — COMMENTED** [review]
there is that one typo.
also why is everything being removed from metrics-catalog.html? (Isn't this generated anyway?)

**Comment by apannetrat** [comment]
I fixed the typo and regenerated the metrics-catalog.html

It seems that generating a file (like metrics-catalog.html) from a fork of a public repo is not possible. See the note [here](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token).
I will add a test to select this in the future in the github workflow.

File: tools/metrics_validator.rb (hunk 1 above)
Comment (pritikin) [reply tools/metrics_validator.rb]: alain fixed in the next commit

**Pull request merged**